classes/handler/APIHandler.php

```diff
@@ -215,6 +225,14 @@ protected function registerRoute(): void
 
                 $route->middleware($middleware);
             }
+
+            // If route has extra policy authorizer from plugin, add those to route's action stack with custom key
+            // which will be later added in the route controller's authorization stack via `PolicyAuthorizer` global middleware
+            if ($routeParams['policyAuthorizer'] && $routeParams['policyAuthorizer'] instanceof HasAuthorizationPolicy) {
+                $route->setAction(array_merge($route->getAction(), [
+                    'policyAuthorizer' => $routeParams['policyAuthorizer']
```

Comment:
Would it make sense to prefix this with `plugin` to make it clearer that it comes from a plugin? That way, it’d be named `pluginPolicyAuthorizer`, which might make its origin more obvious.